docs/source/getting-started.rst

```diff
@@ -137,11 +137,11 @@ Provides a way to fetch balance of ERC20 token specified by ``tokenAddress`` for
 
   export function TokenBalance() {
     const { account } = useEthers()
-    const tokenBalance = useTokenBalance(account, DAI)
+    const tokenBalance = useTokenBalance(DAI, account)
 
     return (
       <div>
-        {tokenBalance && <p>Balance: {formatUnits(tokenBalance, 18)}</p>}
+        {tokenBalance && <p>Balance: {formatUnits(tokenBalance as any, 18)}</p>}
```

Comment:
Why as any?

Reply:
Without it I got an error:
  " Argument of type 'BigNumber' is not assignable to parameter of type 'BigNumberish' "

Reply:
I don't get such error here, do you have another BigNumber library imported in the scope? Maybe other BigNumber library

Reply:
You should be able to fix it by updating to the newest ethers and useDApp.

Reply:
Sometimes, I get such error... But, I try to check properly and most times saw I'm not doing something right

Reply:
There was in the older version problem when app was using ethers 5.1.* and useDApp was using ethers 5.0.*
Latest useDApp version is using 5.1.* and so should your app use. With that it should all work and there should be no type mismatches.